validator/accounts/cli_manager.go

```diff
@@ -21,6 +23,7 @@ import (
 func NewCLIManager(opts ...Option) (*CLIManager, error) {
 	acc := &CLIManager{
 		mnemonicLanguage: derived.DefaultMnemonicLanguage,
+		inputReader:      os.Stdin,
```

Comment:
Is this related to the scope of the PR?

Reply:
yeah based on the reader, I'm not able to mock the input for the read to test the functionality ( it was missing unit tests before)